mathics/builtin/functional/apply_fns_to_lists.py

```diff
@@ -173,21 +173,21 @@ class MapAt(Builtin):
       <dd>applies $f$ to the part of $expr$ at position {$i$, $j$, ...}.
 
       <dt>'MapAt[$f$, $pos$]'
-      <dd>represents an operator form of MapAt that can be applied to an expression.
+      <dd>represents an operator form of 'MapAt' that can be applied to an expression.
     </dl>
 
     Map function $f$ to the second element of an simple flat list:
     >> MapAt[f, {a, b, c}, 2]
      = {a, f[b], c}
 
-    Above we specified a simple integer value 2. In general, the expression can an arbitrary vector.
+    Above, we specified a simple integer value 2. In general, the expression can an arbitrary vector.
```

Comment:
...the expression can `be` an arbitrary vector.

Reply:
Thanks - fixed in 6eacd744